packages/kbn-unified-data-table/src/components/data_table.test.tsx

```diff
@@ -265,34 +297,19 @@ describe('UnifiedDataTable', () => {
 
   describe('edit field button', () => {
     it('should render the edit field button if onFieldEdited is provided', async () => {
-      const component = await getComponent({
-        ...getProps(),
-        columns: ['message'],
-        onFieldEdited: jest.fn(),
-      });
-      expect(findTestSubject(component, 'dataGridHeaderCellActionGroup-message').exists()).toBe(
-        false
-      );
-      findTestSubject(component, 'dataGridHeaderCell-message').find('button').simulate('click');
-      expect(findTestSubject(component, 'dataGridHeaderCellActionGroup-message').exists()).toBe(
-        true
-      );
-      expect(findTestSubject(component, 'gridEditFieldButton').exists()).toBe(true);
+      renderDataTable({ columns: ['message'], onFieldEdited: jest.fn() });
```

Comment:
I needed to rewrite these tests in RTL because Enzyme doesn't properly clean up portals after tests (used for the column header popovers), causing my new tests below to fail.